pkg/operator/ceph/agent/agent.go

```diff
@@ -240,6 +242,14 @@ func (a *Agent) createAgentDaemonSet(namespace, agentImage, serviceAccount strin
 		}
 	}
 
+	var tolerations []v1.Toleration
+	tolerationsRaw := os.Getenv(agentDaemonsetTolerationsEnv)
+	err = k8sutil.YamlToTolerations(tolerationsRaw, &tolerations)
+	if err != nil {
+		logger.Warningf("failed to parse %s", tolerationsRaw)
```

Comment:
Please append the error message, such as:
```
logger.Warningf("failed to parse %s. %+v", tolerationsRaw, err)
```

Reply:
Done.